### DISCUSSION TITLE
Please tell me your interest number

### DISCUSSION THREAD

**Discussion by f-tash** [NAIST-SE/AttractingContributors#1132] [2024 Attraction Contributions]
Please tell me your interest number
repo:https://github.com/f-tash/AttractingContributorsInterestNumber

**Comment by satorukano** [comment]
30 is the most interesting number!
please tell me your hobby.

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Comment by polarbearpppp** [comment]
Hi, I'm contribute to your project , Would you mind to contribute my project
https://github.com/polarbearpppp/AttractingContributors-Games

**Comment by Tz-violet** [comment]
hi, I just suggested my favourite one.
can you help me to complete my project: https://github.com/Tz-violet/AttractingContributors.git ?

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
[https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git](https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git)
Discussion link: [#1148](https://github.com/NAIST-SE/AttractingContributors/discussions/1148)

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！

**Comment by wangzaixiaoliunai** [comment]
I have created a PR.
Could you please contribute to mine?
https://github.com/wangzaixiaoliunai/AttractingContributors-musical_instruments.git

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon
